### ISSUE TITLE
Reproduce result

### ISSUE THREAD

**Issue by thangvubk** [jmiller656/EDSR-Tensorflow#20]
Very good work :D. Btw, did you reproduce the result in the paper?

**Comment by jmiller656** [comment]
Thanks! I haven't evaluated exactly as they did on the paper, but on the test set I used, my network got around (working off of memory, away from my computer atm) ~31 PSNR. These are the results for my pretrained model
